flocx_market/api/app.py

```diff
@@ -37,9 +37,7 @@ def create_app(app_name):
     api.add_resource(
         OfferContractRelationship,
         '/offer_contract_relationship',
-        '/offer_contract_relationship/',
-        '/offer_contract_relationship/'
-        '<string:marketplace_offer_id>/<string:contract_id>')
+        '/offer_contract_relationship/')
```

Comment:
Should there be an `/offer_contract_relationship/<string:offer_contract_relationship_id>` resource here? We kinda just want this to work exactly like offers or contracts or bids; the only difference is that when we "get all", we check for query variables for the filter.

Reply:
yep, responded in previous comment